.github/workflows/mac-artifact.yml

```diff
@@ -1,9 +1,9 @@
 name: Mac Artifact
 on:
   push:
-    branches: [ master, test-ready, "bugfix-*", actionsHub ]
+    branches: [ master, test-ready, "bugfix-*" ]
   pull_request:
-    branches: [ master, test-ready ]
+    branches: [ master, test-ready, "bugfix-*" ]
```

Comment:
I would just check that the builds are not happening twice.

Reply:
I don't see duplicate builds in the actions run logs